src/startCustomTimer.tsx

```diff
@@ -13,6 +13,23 @@ export default function CustomTimerView(props: { arguments: CTInlineArgs }) {
 
   const prefs: Preferences = getPreferenceValues();
 
+  if(hasArgs && prefs.customTimerFormBypass) {
+
+    const [hours,minutes,seconds] =
+      (['hours','minutes','seconds'] as const)
+      .map(k=>props.arguments[k])
+      .map(Number)
+      .map(n=>Number.isNaN(n) ? 0 : n);
+
+    startTimer({
+      timeInSeconds: 3600 * hours + 60 * minutes + seconds,
+      timerName: "Untitled",
+      selectedSound: "default",
```

Comment:
These lines can be dropped, `startTimer` will set these default values automatically.